Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/oryx/build:github-actions-20200706.2
+FROM mcr.microsoft.com/oryx/build:github-actions-20200719.1
```

Comment:
Can you make sure the `v2` tag will point to `master` and also kick off a run from the samples repo after your merge this in?